react-blog-app/src/modals/FormSuccessModal.tsx

```diff
@@ -0,0 +1,26 @@
+import { useHistory } from "react-router-dom";
+
+const FormSuccessModal = (props:any) => {
+
+    const history:any = useHistory();
+
+    const back = (e:any) => {
+        e.stopPropagation();
+        if (history.location.state.background.pathname === "/newarticle") {
+            history.push("/");
+        } else {
+            history.go(-2);
+        }
+      };
```

Comment:
And you wouldn't need this logic if this popup won't have its own address

Reply:
Certainly